### DISCUSSION TITLE
Could not load events - There's a problem with your Seq license

### DISCUSSION THREAD

**Discussion by NightWuYo** [datalust/seq-tickets#1689] [Help] [answered]
Hello team

I'm pretty sure that I'm the only one who is using Seq web UI, and I just opened several tabs in the same chrome instance/profile in the same machine.
I'm wondering how Seq defines "multiple concurrent sessions", is there any way I can share more info so you may look into it?

The error I got:
> There's a problem with your Seq license
> The Individual license only permits one user to access Seq. Queries have been disabled for ten minutes because of multiple concurrent sessions; please close Seq on additional machines or tabs to remove this message and re-enable queries. To try Seq in a team environment, please request a trial license certificate from the Seq website.
> 
> Seq is still collecting application events, but queries will be unavailable until the issue is resolved.

**Comment by nblumhardt** [comment, marked as answer]
Hi! Thanks for reaching out, and sorry about the inconvenience.

The Individual license doesn't support sharing of a Seq server, whether use is concurrent or not, so if this is on a shared machine you'll need a suitable subscription to use Seq there.

If this is on a workstation or a Seq server that only you have access to, it may simply be that you have too many tabs open and have triggered this check. It takes rather **a lot** of tabs, but unfortunately it's hard for us to strike a balance between supporting individual use for free and making sure that the software is properly licensed in a team environment, and sometimes it's possible to hit the limit while using Seq as an individual.

Hope this helps, and please let us know if you need any more info.

Best regards,
Nick

**Reply by NightWuYo** [reply]
Hi Nicholas

Thanks for your quick response.
This is my personal desktop, not a shared one, only I have access to this machine, no remote access.

>It takes rather a lot of tabs

When the issue was triggered, I only have 6 or 7 tabs opened, but since I was testing and creating dashboards, I opened and closed new tabs frequently.
Not sure how many is "a lot", but I don't think I have opened more than 100 tabs in one hour (usually I close new tabs soon, so no more than 10 open tabs at the same time).

I agree that it's hard to get a balance, thank you again.

**Comment by jbccollins** [comment]
This is incredibly obnoxious. How dare I open two browsers with three localhost Seq tabs between them. There has got to be a better way for you to check if someone is abusing the Seq licensing. This developer experience is awful. First day using Seq and I'm ripping it out now.

**Reply by nblumhardt** [reply]
Hi James! Setting an admin password (done automatically by the Windows installer, or using `SEQ_FIRSTRUN_ADMINPASSWORD=somepassword` under Docker) will do this: open tabs are only limited if authentication is not enabled.

Thanks for the feedback, though, we should improve the UX around this 👍
